lib/src/components/StickyNotes/components/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button as WUIButton } from '@/components/Button'
+import type { ButtonProps } from '@/components/Button/types'
+import { classNames } from '@/utils'
+
+import styles from '../sticky-notes.module.scss'
+
+const cx = classNames(styles)
+
+// We force size="sm" and variant="primary-neutral" for consistency and alignment on bottom
+export const Button = ({ children, ...rest }: Omit<ButtonProps, 'size' | 'variant'>) => {
+  return (
+    <WUIButton className={cx('action')} size="sm" variant="primary-neutral" {...rest}>
```

Comment:
If I pass `size="md"` it will override `size="sm"` but from your comment you say you want to hardcode `size="sm"` ?

Reply:
I should be able to pass a `className` as well (to adjust position / margin etc)

Reply:
+1 for the className

But for the size, as the type is `Omit<ButtonProps, 'size' | 'variant'>`, you cannot override it
```
    <StickyNotes>
      <StickyNotes.Title>Title</StickyNotes.Title>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      <StickyNotes.Button onClick={handleClick} size="lg">
        Action
      </StickyNotes.Button>
    </StickyNotes>
```

You have a ts error on the size props ⬇️ 
```
Type '{ children: string; onClick: () => void; size: string; }' is not assignable to type 'IntrinsicAttributes & Omit<ButtonProps, "variant" | "size">'.
  Property 'size' does not exist on type 'IntrinsicAttributes & Omit<ButtonProps, "variant" | "size">'.ts(2322)
```